packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -95,24 +98,20 @@ contract PluginRepo is
         address _pluginSetup,
         bytes calldata _contentURI
     ) external auth(address(this), CREATE_VERSION_PERMISSION_ID) {
-        // Check if `_pluginSetup` is a `PluginSetup` contract
-        if (!Address.isContract(_pluginSetup)) {
-            revert InvalidContractAddress({invalidContract: _pluginSetup});
+        // In a case where _pluginSetup doesn't contain supportsInterface,
+        // but contains fallback, that doesn't return anything(most cases)
+        // the below approach aims to still return custom error which not possible with try/catch..
```

Comment:
Don't we want to keep checking that Plugin Setup is a contract?

Reply:
The `functionCall` checks it implicitly.